namespaces/live.cloud-platform.service.justice.gov.uk/care-arrangement-plan-prototype/resources/variables.tf

```diff
@@ -0,0 +1,76 @@
+variable "vpc_name" {
+  description = "VPC name to create security groups in for the ElastiCache and RDS modules"
+  type        = string
+}
+
+variable "kubernetes_cluster" {
+  description = "Kubernetes cluster name for references to secrets for service accounts"
+  type        = string
+}
+
+variable "application" {
+  description = "Name of the application you are deploying"
+  type        = string
+  default     = "Care Arrangement Plan"
+}
+
+variable "namespace" {
+  description = "Name of the namespace these resources are part of"
+  type        = string
+  default     = "care-arrangement-plan-prototype"
+}
+
+variable "business_unit" {
+  description = "Area of the MOJ responsible for this service"
+  type        = string
+  default     = "HQ"
+}
+
+variable "team_name" {
+  description = "Name of the development team responsible for this service"
+  type        = string
+  default     = "central-digital-product-team"
+}
+
+variable "environment" {
+  description = "Name of the environment type for this service"
+  type        = string
+  default     = "development"
+}
+
+variable "infrastructure_support" {
+  description = "Email address of the team responsible this service"
+  type        = string
+  default     = "central-digital-product-team@justice.gov.uk"
+}
+
+variable "is_production" {
+  description = "Whether this environment type is production or not"
+  type        = string
+  default     = "false"
+}
+
+variable "slack_channel" {
+  description = "Slack channel name for your team, if we need to contact you about this service"
+  type        = string
+  default     = "cdpt-care-arrangement-plan"
+}
+
+variable "github_owner" {
+  description = "The GitHub organization or individual user account containing the app's code repo. Used by the Github Terraform provider. See: https://user-guide.cloud-platform.service.justice.gov.uk/documentation/getting-started/ecr-setup.html#accessing-the-credentials"
+  type        = string
+  default     = "ministryofjustice"
+}
+
+variable "github_token" {
+  type        = string
+  description = "Required by the GitHub Terraform provider"
+  default     = ""
+}
+
+## Prototype kit variables
+
+variable "password" {
```

Comment:
I don't think we have a mechanism to securely get a password into terraform this way.

Is this going to be a password that is available to the application as an env var?

If so, could you add it as a GitHub secret and we can handle it as part of the application deployment in the application repo.

Reply:
Yep, its a password used to access the prototype (doesn't need to be secure, its just to stop people randomly stumbling across it).

I was initially going to add it as a Github actions secret, but saw this was how it was set up in the cloud platform prototype setup.

I'll move it to the prototype repo - I agree it makes much more sense!